src/styles/blocks/card.scss

```
@@ -0,0 +1,137 @@
+.container {
+  display: grid;
+  justify-content: center;
+  grid-gap: 46px 48px;
+  padding: 50px 40px;
+
+  @include number-of-grid-template-columns;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  font-family: inherit;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
```

don't forget to create variables for all colors
```suggestion
  border: 1px solid ....;
```